content/docs/command-reference/exp/apply.md

```diff
@@ -14,22 +14,25 @@ positional arguments:
 
 ## Description
 
-Restores an `experiment` into the workspace as long as no more Git commits have
-been made after the target experiment (`HEAD` hasn't moved). The `experiment`
-can be referenced by name or hash (see `dvc exp run` for details). This changes
-any files (code, data, <abbr>parameters</abbr>, <abbr>metrics</abbr>, etc.)
-needed to reflect the experiment conditions and results in the workspace.
+Restores an `experiment` into the workspace, as long as we're on the same
+project baseline (Git `HEAD`) as when the target experiment was run. The
+experiment can be referenced by name or hash (see `dvc exp run` for details).
 
-⚠️ Conflicting changes in the workspace are overwritten unless `--no-force` is
-used.
+Specifically, `dvc exp apply` checks out any files or directories needed to
+reflect the experiment conditions and results. This can include both with DVC
+and Git: code, data, <abbr>parameters</abbr>, <abbr>metrics</abbr>, etc.
```

Comment:
I think the previous expression is more clear.

Reply:
I don't love the 2nd sentence here either if that's what you mean @iesahin. But the previous wording was a single long sentence instead of 2 which mixed the 2 ideas so it was harder to read and comprehend IMO. I also changed "restores" for "check out" to be more specific about the operation what takes place.

Reply:
p.s. I think I clarified the wording now (817d6eee).